package.json

```diff
@@ -26,7 +28,7 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "@fortawesome/fontawesome": ">=1.2.0-5",
+    "@fortawesome/fontawesome-svg-core": "^1.2.0-8",
```

Comment:
Just want to double-check that this doesn't rather need to be in `peerDependencies` instead. But I'm having a hard time thinking it through. Since we walked through this on the Angular component, could you consider whether that applies in this case as well?

Reply:
Ah. Yes. Put that in “peerDependencies” and in “devDependencies” (so tests will work). It should not be in “dependencies”.

Reply:
Just circling back around here to close the loop from our Slack convo: We decided to leave this in `dependencies`.